Posit Infrastructure/Guide to using renv to create reproducible projects.md

```diff
@@ -0,0 +1,47 @@
+# Guide to using [{renv}](https://rstudio.github.io/renv/) to create reproducible projects
+
+## Purpose
+
+Our code should be reproducible and shareable. But sometimes things fail to run properly when we return to them, or on a colleague's machine. The R package [{renv}](https://rstudio.github.io/renv/) ensures that the exact set of packages (and their versions) required to run the code in a specific project are recorded and kept available to current and future users. 
+
+This guide provides basic step-by-step instructions for using [{renv}](https://rstudio.github.io/renv/), as well as more detailed information. 
+
+## Step-by-step instructions
+
+Pre-requisite: a project (preferably version-controlled using git).
+
```

Comment:
Should we include a section on creating a brand new project with renv enabled from the start, as this can be done via the RStudio IDE.